### ISSUE TITLE
Please change external links interaction to opening from a new page

### ISSUE THREAD

**Issue by shanyi15** [PaddlePaddle/PaddlePaddle.org#433]
According to Cheradm's comment, please change external links interaction to opening from a new page.
For example:
Click "ADADELTA: AN ADAPTIVE LEARNING RATE METHOD" here
http://www.paddlepaddle.org/docs/develop/api/en/v2/config/optimizer.html#permalink-6-adadelta
![image](https://user-images.githubusercontent.com/35982308/36955438-81e83f2a-2063-11e8-9a5b-97a0fa6df366.png)
the default open function of external link is opening a new page.
![image](https://user-images.githubusercontent.com/35982308/36955511-de848e14-2063-11e8-9d2d-bd8721f99b2b.png)

Thank you!

**Comment by daming-lu** [comment]
Users should decide if it is better to open the link in new tab/page or in the current page. In RST, there is no easy way to specify this behavior. Internal links should open in the current page whereas external page could be opened in new tab. But for now there is no easy way to distinguish them. See a post [here](https://stackoverflow.com/questions/11716781/open-a-link-in-a-new-window-in-restructuredtext).

**Comment by shanyi15** [comment]
had taken a look at pytorch and tensorflow, clicking a link won't open a new page.
